_projects/sssp_project.md

```diff
@@ -79,7 +83,6 @@ Remember to use the `--file jlesc.bib` with the `cite` tag.
 
 ## Future plans
 
-See above.
 
```

Comment:
Are there any future plans? Is the project finished? If so, please change the status. If not, please add a line here.

Reply:
I've added the plan.

Reply:
Still need to submit, I can't see it online. The same goes for the reference fix in the text above.